pyaedt/application/Analysis3D.py

```diff
@@ -1277,6 +1277,117 @@ def import_dxf(
         self.oeditor.ImportDXF(vArg1)
         return True
 
+    @pyaedt_function_handler
+    def import_gds_3d(self, gds_file_path, gds_number_list, elevation_list=None, thickness_list=None, import_method=1):
+        """Import a GDSII file.
+
+        Parameters
+        ----------
+        gds_file_path : str
+            Path to the GDS file.
+        gds_number_list : list
+            List of layer number to import.
+        elevation_list : list, optional
+            List of elevation for each GDS layer number imported.
+            Number of element must be the same as gds_number list.
+            If not or None elevation and thickness will not be applied.
```

Comment:
@amichel0205 The default is ````None```` in which case the thickness won't be applied.

Reply:
Is it better like this;
        elevation_list : list, optional
            List of GDS layer elevation.
            Number of element must be the same as gds_number list, if not elevation and thickness will not be applied.
            If elevation is none, elevation and thickness will not be applied.